packages/compass-aggregations/src/components/stage-preview/stage-preview.jsx

```diff
@@ -111,7 +111,7 @@ class StagePreview extends Component {
     if (this.props.isComplete) {
       if (!this.props.error) {
         return (
-          <Body className={styles['stage-preview-out']}>
+          <Body as="div" className={styles['stage-preview-out']}>
```

Comment:
We should open a PR to make this the default as you suggested somewhere else .. is super easy to forget